src/Miki/resources/locales/eng.json

```diff
@@ -123,9 +123,9 @@
   "miki_module_pasta_error_search_no_results": "No search query found. Please specify what you want to search for.",
   "miki_module_pasta_error_specify": "Please specify which pasta you would like to {0}.",
   "miki_module_pasta_error_specify_check": "check",
-  "miki_module_pasta_error_specify_delete": "delete",
-  "miki_module_pasta_error_specify_edit": "edit",
-  "miki_module_pasta_error_specify_upvote": "upvote",
+  "verb_delete": "delete",
```

Comment:
Not sure between either Verb or Action

Reply:
Verb looks good to me, but either way, it gets the message across. I like the shortened format